docs/tutorials/point_source.ipynb

```diff
@@ -162,11 +163,11 @@
    "outputs": [],
    "source": [
     "# Initialize the Blend object, which later fits the model\n",
-    "blend = scarlet.Blend(scene, sources, observation)\n",
+    "blend = scarlet.Blend(frame, sources, observation)\n",
     "\n",
     "# Display the initial model\n",
-    "_model = blend.get_model()\n",
-    "model = observation.get_model(_model)\n",
+    "model = blend.get_model()\n",
+    "model_ = observation.render(model)\n",
```

Comment:
Something isn't right. It doesn't appear to be properly matching the observation PSFs, causing the output to be in the model frame, not the observation frame. Compare the results of this notebook with https://fred3m.github.io/scarlet/tutorials/point_source.html#Fit-the-model-and-display-the-results

Reply:
I found the bug. In both this cell and the next one you use `model`, not `model_`.

Reply:
Actually, it's probably better to use
```
_model = blend.get_model()
model = observation.render(_model)
```
so that you only have to make one change. Otherwise you not only have to change the way you create the RGB images, you also have to modify the line where you calculate the residual.

Reply:
No problem, but I prefer it this way: `model` is direct from `Blend`, `model_` is after it gets mangled by `render`